WordPress/Classes/ViewRelated/People/InvitePersonViewController.swift

```diff
@@ -363,6 +368,14 @@ private extension InvitePersonViewController {
         // By default, Send is disabled
         navigationItem.rightBarButtonItem?.enabled = false
     }
+
+    func setupDefaultRole() {
```

Comment:
I was thinking that maybe we can replace this method with a property initializer.  It's pretty minor but I believe that could be a nice change.  Thoughts?